frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -1,18 +1,107 @@
 <template>
   <div>
-    <span>{{ contests }}</span>
+    <div class="col-sm-12">
+      <h1 class="title">{{ T.wordsContests }}</h1>
+    </div>
+    <b-card no-body>
+      <b-tabs
+        v-model="activeTab"
+        class="sidebar"
+        pills
+        card
+        vertical
+        nav-wrapper-class="custom-nav col-sm-4 col-md-2"
```

Comment:
```suggestion
        nav-wrapper-class="contest-list-nav col-sm-4 col-md-2"
```

"custom" puede significar un montón de cosas. mejor hay que usar algo que indique su propósito.